checkpoint/runner_test.go

```diff
@@ -33,12 +33,12 @@ func TestMain(m *testing.M) {
 var allAtxs = map[types.NodeID][]*types.ActivationTx{
 	// smesher 1 has 7 ATXs, one in each epoch from 1 to 7
 	types.BytesToNodeID([]byte("smesher1")): {
-		newAtx(types.ATXID{17}, types.ATXID{16}, nil, 7, 6, 0, types.BytesToNodeID([]byte("smesher1"))),
-		newAtx(types.ATXID{16}, types.ATXID{15}, nil, 6, 5, 0, types.BytesToNodeID([]byte("smesher1"))),
-		newAtx(types.ATXID{15}, types.ATXID{14}, nil, 5, 4, 0, types.BytesToNodeID([]byte("smesher1"))),
-		newAtx(types.ATXID{14}, types.ATXID{13}, nil, 4, 3, 0, types.BytesToNodeID([]byte("smesher1"))),
-		newAtx(types.ATXID{13}, types.ATXID{12}, nil, 3, 2, 0, types.BytesToNodeID([]byte("smesher1"))),
-		newAtx(types.ATXID{12}, types.ATXID{11}, nil, 2, 1, 0, types.BytesToNodeID([]byte("smesher1"))),
+		newAtx(types.ATXID{17}, types.ATXID{16}, nil, 7, 6, 123, types.BytesToNodeID([]byte("smesher1"))),
```

Comment:
Same here, if we make `ID` public (which we in my opinion should) the `newAtx` function isn't needed any more.